### PR TITLE
TER-103 revoke bucket read only role

### PR DESCRIPTION
**Pull request by martinjunger**
Jira: TER-103
driver-teradata: https://github.com/keboola/php-storage-driver-teradata/pull/60

### CODE REVIEW

File: proto/bucket.proto [hunk 1]
```diff
@@ -98,3 +98,10 @@ message GrantBucketAccessToReadOnlyRoleCommand {
   string projectReadOnlyRoleName = 2;
   google.protobuf.Any meta = 3;
 }
+
+message RevokeBucketAccessFromReadOnlyRoleCommand {
+  string bucketObjectName = 1; // object name in backend
+  string projectReadOnlyRoleName = 2; // backend read only role associated with project
+  bool ignoreErrors = 3; // if true all backend errors should be ignored and command will always pass
```
Comment (jirkasemmler) [inline proto/bucket.proto:105]: proc to potrebujeme? nemel by ten ignore byt v connection?

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
jsi evidentne zmenil vsude chmod, prosim fixni to

**Comment by martinjunger** [comment]
> jsi evidentne zmenil vsude chmod, prosim fixni to

znovu jsem zbuildil `dev` container a pustil `composer protobuf` a pomohlo to - stejne jako predtim 🤷

File: proto/bucket.proto (hunk 1 above)
Comment (jirkasemmler) [reply proto/bucket.proto:105]: dava to smysl na vicekrokovych operacich -> ergo by to melo byt i tady

**Review by jirkasemmler — APPROVED** [review]

**Pull request merged**